### ISSUE TITLE
fribidi download source is gone

### ISSUE THREAD

**Issue by Fuuzetsu** [NixOS/nixpkgs#32709]
## Issue description
fribidi.org is no longer available and redirects to github; the download URL 404s

### Steps to reproduce
Try to build fribidi on 1a98fc40e9bbfe6bdf17930b5fa926b3f50e5d27 ; I notice that you can get lucky and get it from nix cache but it should be updated

## Technical details
Sadly just doing 

```
-  src = fetchurl {
-    url = "http://fribidi.org/download/${name}.tar.bz2";
-    sha256 = "13jsb5qadlhsaxkbrb49nqslmbh904vvzhsm5mm2ghmv29i2l8h8";
+  src = fetchFromGitHub {
+    owner = "fribidi";
+    repo = "fribidi";
+    rev = version;
+    sha256 = "10q5jfch5qzrj2w4fbkr086ank66plx8hp7ra9a01irj80pbk96d";
   };
```

didn't seem enough: it can't find `aclocal`, there's no configure file out of the box as it expects… I don't know why it's different from the previous tarball.

**Comment by dtzWill** [comment]
Add `autoreconfHook` to nativeBuildInputs?

As mentioned on the graphviz issue, git exports often lack generated files like `configure`.

If this works, please submit it as a PR!

**Comment by yegortimoshenko** [comment]
This has been fixed in #32725.
